### ISSUE TITLE
libxml2 library not found

### ISSUE THREAD

**Issue by drpventura** [flycheck/flycheck#491]
I am getting the above message after installing flycheck using package.el in emacs 24.3.1 on Windows 7 64-bit, even on test.js.

When I try my own file:

``` javascript

var obj = {};

docu;
```

in addition to the libxml2 library not found I also get the following message:

``` xml
Checker javascript-jshint returned non-zero exit code 2, but no errors from output: <?xml version="1.0" encoding="utf-8"?>
<checkstyle version="4.3">
    <file name="c:/Users/Lope/AppData/Local/Temp/flycheck4704WpI/deleteme.js">
        <error line="4" column="1" severity="warning" message="Expected an assignment or function call and instead saw an expression." source="jshint.W030" />
    </file>
</checkstyle>

Checker definition probably flawed.
```
